bricks/vs/src/code-editor/workers/parseYaml.worker.ts

```diff
@@ -0,0 +1,164 @@
+import type { MemberExpression, Identifier } from "@babel/types";
+import type * as monaco from "monaco-editor/esm/vs/editor/editor.api.js";
+import BrickNextYamlSourceMap, { Token } from "../utils/brickNextSourceMap.js";
+import { Marker } from "../index.jsx";
```

Comment:
主动标识下只引入类型，不然后面意外加个 import 值就可能造成 worker 过大。

```suggestion
import type { Marker } from "../index.jsx";
```